lib/grinder_tools.dart

```diff
@@ -175,6 +176,58 @@ class Tests {
   }
 }
 
+/// A wrapper around the `test` package. This class is used to run your unit
+/// tests.
+class Test {
+  final PubApp _test = new PubApp.local('test');
+
+  Test();
+
+  /// Run the tests in the current package. See
+  /// https://pub.dartlang.org/packages/test.
```

Comment:
will dartdoc make this link clickable?  Maybe use a markdown link:

the [test package](https://pub.dartlang.org/packages/test).